test/xpu_api/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -80,30 +80,25 @@ test(typename std::enable_if<!std::is_integral<T>::value>::type* = 0, typename s
 
 ONEDPL_TEST_NUM_MAIN
 {
+#if !TEST_OF_COMPLEX_POW_BROKEN
     test<int, float>();
+#endif // !TEST_OF_COMPLEX_POW_BROKEN
+
     test<unsigned, float>();
     test<long long, float>();
 
-    auto fnc1 = []()
-    {
-        test<int, double>();
-        test<unsigned, double>();
-        test<long long, double>();
-        test<float, double>();
-        test<double, float>();
-    };
-    IF_DOUBLE_SUPPORT(fnc1);
-
-    auto fnc2 = []()
-    {
-        test<unsigned, long double>();
-        test<long long, long double>();
-        test<float, long double>();
-        test<double, long double>();
-        test<long double, float>();
-        test<long double, double>();
-    };
-    IF_LONG_DOUBLE_SUPPORT(fnc2);
+    IF_DOUBLE_SUPPORT(test<int, double>();
```

Comment:
Change in this line please see in https://github.com/oneapi-src/oneDPL/pull/667
This change is also exist in this PR only to avoid other compile errors.